app/models/ingredient.rb

```diff
@@ -0,0 +1,6 @@
+class Ingredient < ApplicationRecord
+  validates :name, presence: true,
+    uniqueness: true,
+    format: { with: /\A[a-z][a-z\s]+[a-z]\z/,
+    message: "only allows underscore letters and whitespace" }
```

Comment:
it is not underscore, but lowercase letters. Underscore is a `_` character. Do you really need a string containing only letters? Maybe validation against presence would be enough?